### ISSUE TITLE
Document structure of ".terms" file

### ISSUE THREAD

**Issue by wtbarnes** [rice-solar-physics/ebtelPlusPlus#54]
ebtel++ can optionally output a `.terms` file that saves the conductive flux, radiative loss, and c_1 terms. The structure of this should be documented similar to the other files in the docs.

**Comment by wtbarnes** [comment]
After #98, this is no longer relevant since results are not returned as individual results files.
